src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -341,7 +341,8 @@ protected void onShutdown() {
 
         if ( engine != null ) {
             logger.info(String.format("***Processed %d total sites", engine.getTotalNumberOfSites()));
-            logger.info(String.format("***Processed %d total variants", engine.getTotalNumberOfVariants()));
+            logger.info(String.format("***Processed %d total variants", engine.getTotalNumberOfVariants()));            logger.info(String.format("***Processed %d total variants", engine.getTotalNumberOfVariants()));
```

Comment:
this formatting looks kind of odd--maybe it's just github?

Reply:
oh yeah that is strange…

Reply:
fixed